test/spec/controllers/home.ctrl.spec.js

```diff
@@ -7,14 +7,16 @@ describe('HomeCtrl', function() {
     module('yourStlCourts');
 
     inject(function($controller) {
-      HomeCtrl = $controller('HomeCtrl', {});
+      HomeCtrl = $controller('HomeCtrl', function($state,Courts,municipalities){
+
+      });
```

Comment:
This syntax is incorrect. Look at the other controller spec files for an example. Should be no "function(...."